Essentials/src/com/earth2me/essentials/utils/FormatUtil.java

```diff
@@ -54,11 +55,7 @@ public static String placeholderAPIFormat(final Player player, String input) {
             return null;
         }
 
-        input = input.replaceAll("\\{|\\}", "%");
-
-        if (papiEnabled == null) {
```

Comment:
This still needs to be here, in case PAPI loads before EssentialsX - in that instance, Essentials won't receive PAPI's `PluginEnableEvent`.

Reply:
That is also the reason for the null - at the point that the class is loaded, we won't know whether PAPI is enabled, and it also might not be a safe time to check.

Reply:
Ah, that make sense.  Let me change that

Reply:
Tbh, we can put false rather than null. If PluginEvent didn't fired while PAPI excistance still there then it would still false anyway. `false` seem more appropiate this time.